### DISCUSSION TITLE
Assistants API Integration

### DISCUSSION THREAD

**Discussion by nicholastrofin** [567-labs/instructor#345] [General]
Hello everyone!
First of all, thank you Jason for creating this stunning project and thank you to everyone for supporting this poject. The reason I want the Assistants API Integration is because I would like to give a bot specific instructions, say to be a math tutor. However I want it to only give exact information and use the validator built in the instructor library. It is however highly important that I give it specific instructions before asking it questions, but I do not want to do it everytime I have to send a message to it. 

Now I am not set on having to use the Assistants API but I don't know of any other way to prompt the chat in the background.

What is your solution to this?

**Comment by UncleDrew92** [comment]
I'm also curious how to work with assistants and instructor.

**Comment by jxnl** [comment]
It's not really a priority, honestly.
